static/templates/subscription_settings.hbs

```diff
@@ -29,18 +25,16 @@
         </div>
         <div class="stream-description">
             <span class="stream-description-editable editable-section description rendered_markdown" data-no-description="{{t 'No description.' }}">{{rendered_markdown rendered_description}}</span>
-            {{#if can_change_name_description}}
-            <span class="editable" data-make-editable=".stream-description-editable"></span>
+            <span class="editable stream-settings-header-controls {{#unless can_change_name_description}}hide{{/unless}}" data-make-editable=".stream-description-editable"></span>
             <span class="checkmark" data-finish-editing=".stream-description-editable">✓</span>
-            {{/if}}
         </div>
         <div class="subscription-type">
             <div class="subscription-type-text">
                 {{> subscription_type
                   stream_post_policy_values=../stream_post_policy_values
                   message_retention_text=../message_retention_text}}
             </div>
-            <a class="change-stream-privacy" {{#unless can_change_stream_permissions}}style="display: none;"{{/unless}}>[{{t "Change" }}]</a>
+            <a class="change-stream-privacy {{#unless can_change_stream_permissions}}hide{{/unless}}">[{{t "Change" }}]</a>
```

Comment:
A lot of this refactoring looks like it could be done in prep commits that just move us to declarative variables that initially still equal `is_realm_admin`; I think that'd be well worth doing to reduce the complexity of this commit.